docs/en/operation/Metrics-System.md

```diff
@@ -185,18 +185,18 @@ Detailed descriptions of those metrics are in [cluster metrics]({{ '/en/referenc
 
 `Saved Under FileSystem Operations` shows the operations fulfilled by Alluxio's namespace directly
 without accessing UFSes. Performance improvement can be significant if the target UFS is remote or slow in response.
-Costs can be saved if the underlying storage charges based on requests.
+Costs can be saved if the underlying storage charges are based on requests.
 
 ## Grafana Web UI with Prometheus
 
-Grafana is a metics analytics and visualization software used for visualizing time series data. You can use Grafana to better visualize the various metrics that Alluxio collects. The software allows users to more easily see changes in memory, storage, and completed operations in Alluxio.
+Grafana is a metrics analytics and visualization software used for visualizing time series data. You can use Grafana to better visualize the various metrics that Alluxio collects. The software allows users to more easily see changes in memory, storage, and completed operations in Alluxio.
 
-Since Grafana has a well support to Prometheus, following steps can help you to build your Alluxio monitoring based on Grafana and Prometheus easily.
+Since Grafana has well support to Prometheus, the following steps can help you to build your Alluxio monitoring based on Grafana and Prometheus easily.
```

Comment:
uhh i dont really know the relationship between grafana and prometheus, but neither of these are correct...
from reading https://prometheus.io/docs/visualization/grafana/, maybe `Since Grafana supports visualizing data from Prometheus, the following steps can...`
or omit the section completely and start with `The following steps can...`

Reply:
Your line sounds much better... much cleaer